Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoginBean.java

```diff
@@ -72,7 +72,7 @@
      */
     public Boolean isCorrect = false;
 
-    private static boolean firstTimeLogged = true;
+    private boolean firstTimeLogged = true;
```

Comment:
Can you move the associated logic to happen once per application (during initialization)?